packages/katello/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -86,6 +86,11 @@ cp -pa $RPM_BUILD_DIR/%{gem_name}-%{version}/config/config.yml.example %{buildro
 %foreman_bundlerd_file
 %foreman_precompile_plugin -s
 
+# This plugin is not able to compile static assets directly
+mkdir -p %{buildroot}/%{rubygem_redhat_access_dir}/public/assets
+cp -r  $RPM_BUILD_DIR/%{gem_name}-%{version}/vendor/assets/images/*  %{buildroot}/%{rubygem_redhat_access_dir}/public/assets
```

Comment:
The `rubygem_redhat_access_dir` macro is undefined. You can see that in the build error as well:

```
error: Installed (but unpackaged) file(s) found:
   /%{rubygem_redhat_access_dir}/public/assets/insights/Icon_RH_Document_Blueprint_RGB_Button.png
```